### PR TITLE
Added the manager as dependency, updated demo with form to create stages

### PR DESCRIPTION
**Pull request by MrFastDie**
In this PR I did a hell ton of things.

Most important change is that its compiled down to static JS including the manager to create brackets (storage is the local storage in the browser atm - so that we don't need another service running)

It's an extended example of how to create `round_robin`, `single_elimination` and `double_elimination` stages.

Furthermore, it exports the lang module and used 'en' language as fallback if in the native language the key is not found instead of throwing a compilation error.

### CODE REVIEW

File: src/i18n/index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+"use strict";
```
Comment (Drarig29) [inline src/i18n/index.js:1]: I would prefer not having compiled JS files in the repo (other than in the dist folder).

File: src/i18n/index.ts [hunk 2]
```diff
@@ -1,7 +1 @@
-import en from './en/translation.json';
-import fr from './fr/translation.json';
```
Comment (Drarig29) [inline src/i18n/index.ts:1]: Why did you remove the content of this file?

**Review by Drarig29 — CHANGES_REQUESTED** [review]

File: src/i18n/index.ts (hunk 2 above)
Comment (MrFastDie) [reply src/i18n/index.ts:1]: I moved it to the lang.ts

File: src/i18n/index.js (hunk 1 above)
Comment (MrFastDie) [reply src/i18n/index.js:1]: Ah they were a mistake anyways I tested smth with the code from https://github.com/Drarig29/brackets-manager.js/issues/48#issuecomment-713010349

**Review by Drarig29 — APPROVED** [review]
